wapitiCore/attack/mod_check_cookies_headers.py

```diff
@@ -0,0 +1,43 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+
+class mod_check_cookies_headers(Attack):
+
+    name = "check_cookies_headers"
+
+    def check_secure_flag(self, cookie: object):
+
+        return cookie.secure
+
+    def check_httponly_flag(self, cookie: object):
+
+        return bool('HttpOnly' in cookie._rest)
+
+    def attack(self):
+        url = self.persister.get_root_url()
+        session = requests.session()
+        _req = session.get(url)
+        request = Request(url)
+        for cook in session.cookies:
+            self.log_blue('Check HttpOnly flag :')
+            if not self.check_httponly_flag(cook):
+                self.log_red('HTTPONLY FLAG IS NOT SET')
```

Comment:
You should use constant strings with _("...") notation to support translations + remove CAPS

Reply:
Fixed